react/lib/tests/number.test.ts

```diff
@@ -0,0 +1,35 @@
+import BigNumber from 'bignumber.js';
+import { zero, resolveNumber, isLessThanZero } from '../util/number'; // Adjust the import path
+
+describe('Number util functions', () => {
+  it('zero should be an instance of BigNumber and equal to 0', () => {
+    expect(zero).toBeInstanceOf(BigNumber);
+    expect(zero.isEqualTo(0)).toBe(true);
+  });
+
+  it('resolveNumber should convert various types to BigNumber', () => {
```

Comment:
let's tests floats too (42.0, 42.5) and string floats ('42.0', '42.5'). Also, they shouldn't be in the same tests